uiowa/uiowa
# Tippie Redirects import

Resolves #5265 





# How to test



- `git checkout tippie-redirects && git pull`
- `ddev blt site=tippie.uiowa.edu`
- Watch that tippie core update hook 9001 runs.
- some redirects will be deleted if they form redirects to themselves, `/alumni` was one of these such redirects. Check to make sure it is functioning.
- Spot check the listed redirects, check number of redirects in db before and after update hook, and/or any other method of checking that the correct number of redirects have been created
